packages/cli/src/config/index.ts

```diff
@@ -122,7 +122,7 @@ if (executionProcess === 'own') {
 }
 
 setGlobalState({
-	defaultTimezone: config.getEnv('generic.timezone'),
+	defaultTimezone: Container.get(GlobalConfig).generic.timezone,
```

Comment:
At some point we should also stop using `GlobalConfig` everywhere. Every config class is already individually injectable.